### PR TITLE
Week out mail, schedule tweaks

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: docs/conf/eu/2016/news/welcome.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+We have some reminders, late-breaking information, and a few thanks are in order.
+
+We hope you're getting excited!
+
+## Info
+
+* If you have signed up for the **boat trip**, join us on Saturday the 17th at Prague Boats,
+  [pier no. 5 of Dvořákovo nábřeží](https://goo.gl/maps/bqLP3VaytVo). Boarding is
+  at 13:45, don't be late! The trip lasts for two hours, and will feature an  
+  English-speaking guide.
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/welcome.md]: this line seems to break into a newline?

File: docs/conf/eu/2016/lightning-talks.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
+:template: 2016/eu.html
+
+.. _speaker-lightning-talks:
+
+Lightning Talks
```
Comment (thatdocslady) [inline docs/conf/eu/2016/lightning-talks.rst:5]: awesome! <3

File: docs/conf/eu/2016/news/welcome.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+* **Join us on Sunday** for the [Writing Day](http://www.writethedocs.org/conf/eu/2016/writingday/) from *09:00 to 17:00*, immediately followed by the reception from *17:00-19:00*. You can pick up your badge and enjoy a couple of drinks on us. This is a great chance to meet your fellow documentarians before the conference starts.
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/welcome.md]: please add that the writing day includes breakfast and lunch, and point them to the schedule for details :)

File: docs/conf/eu/2016/news/welcome.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+* **Plan a discussion**. We'll be running the [Unconference](http://www.writethedocs.org/conf/eu/2016/unconference/) in the afternoon on Monday and Tuesday. Want to chat with like-minded folks about a very specific thing? This is the place. We'll have a sign-up board so you know what's happening, and supplies to help you brainstorm.
+* **Come to the party on Monday night.** This is a private party just for conference attendees. Plan to attend so you can get to know folks in the community. We'll be at [Club Lavka](https://goo.gl/maps/3k5XZQvkHZr), our awesome conference venue from last year! Party time is 19:00-23:00, light snacks and various beverages provided.
+* **Coffee, breakfast, and lunch are provided.** Don't worry about coffee before you come to the conference on Monday or Tuesday. Coffee and breakfast are served at 09:00, and the conference starts at 10:00.
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/welcome.md]: also on writing day, and we can write the vegetarian and gluten free documentarians will be taken care of :)

**Comment by plaindocs** [comment]
@thatdocslady @ericholscher anything to add or correct? Speak now or forever hold your peace...

**Comment by ericholscher** [comment]
Looks good!

**Pull request merged**